docs/devops.md

```diff
@@ -534,13 +534,13 @@ npm run create:config && npm run build:curriculum && npm run build:server
 4. Start Instances
 
 ```console
-pm2 start all --update-env && pm2 logs
-```
+   cd api-server && pm2 reload ecosystem.config.js cd .. && pm2 logs
```

Comment:
```suggestion
   cd api-server && pm2 start ecosystem.config.js && cd .. && pm2 logs
```

Reply:
[Do you even look at your own changes?](https://github.com/freeCodeCamp/freeCodeCamp/pull/49379/files#diff-b4e944b268ad7c01bdcd9d3e05f76e12718cbd446e8ed55447145b92daca61d6L488-L494)

Reply:
Does it matter? The main issue was the missing `&&`.

Reply:
🤦🏽‍♂️